scripts/ci/tags.yaml

```diff
@@ -644,3 +644,30 @@ ci_tests_subsys_app_event_manager:
     - nrf/tests/subsys/app_event_manager/
     - nrf/subsys/app_event_manager/
     - nrf/include/app_event_manager.h
+
+ci_applications_machine_learning:
+  files:
+    - nrf/applications/ipc_radio/
+    - nrf/applications/machine_learning/
+    - nrf/drivers/mpsl/
+    - nrf/drivers/sensor/sensor_sim/
+    - nrf/lib/edge_impulse/
+    - nrf/lib/wave_gen/
+    - nrf/samples/nrf5340/netboot/
+    - nrf/subsys/app_event_manager/
+    - nrf/subsys/bluetooth/
+    - nrf/subsys/bootloader/
+    - nrf/subsys/caf/
+    - nrf/subsys/event_manager_proxy/
+    - nrf/subsys/mpsl/
+    - nrf/subsys/pcd/
+    - zephyr/drivers/sensor/adi/adxl362
```

Comment:
This sample needs sensors. Sensors are "connected" with SHIELD pca63565 and pca63566.
What about adding to the list files at:
nrf/boards/shields/pca63565
nrf/boards/shields/pca63566
https://github.com/nrfconnect/sdk-nrf/tree/main/boards/shields/pca63565

Reply:
Thanks, added.